docs/releases.md

```diff
@@ -6,10 +6,122 @@ sidebar_label: Releases
 
 ------
 
-## 2.7.0 - Mar 16 2021
+## 2.9.0 - May 15 2021
 
 <br><font size="4">Latest Release</font><br/> (Recommended)<br/>
 
+OpenEBS v2.9 is another maintenance release before moving towards 3.0 primarily focusing on enhancing the E2e tests and build/release workflows. This release includes fixes for user-reported critical bugs as well as fixes and enhancements to improve the E2e test coverage. There has been some significant progress made on the alpha features as well. 
+
+
+### Key Improvements
+
+- Enhanced [ZFS Local PV](https://github.com/openebs/zfs-localpv) to use a custom node label called `openebs.io/nodeid` to set the node affinity for the provisioned volume. By default, the value will be the same as `kubernetes.io/hostname`.  Using a custom label like this will help in quickly migrating the volumes to a new node in cases where a node fails and the user needs to move the underlying disks to a new node in the cluster. After moving the disks, the user can set the `openebs.io/nodeid` with the value used in the previous node. (https://github.com/openebs/zfs-localpv/issues/304). You can read more about this feature [here](https://github.com/openebs/zfs-localpv/blob/master/docs/faq.md#8-how-to-migrate-pvs-to-the-new-node-in-case-old-node-is-not-accessible).
+- Enhanced [cStor Velero plugin](https://github.com/openebs/velero-plugin/pull/154) to allow users to specify a custom timeout for completing snapshot operations. The time out can be configured via the `restApiTimeout ` field in VolumeSnapshotLocal. See [example](https://github.com/openebs/velero-plugin/blob/master/example/06-volumesnapshotlocation.yaml). (https://github.com/openebs/velero-plugin/issues/148)
```

Comment:
timeout or time out?

Reply:
timeout is valid.

Reply:
There is one time out. This can be timeout then...